msal/src/Microsoft.Identity.Client.Ref/Microsoft.Identity.Client.Ref.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+
+  <!-- This project is used to generated a ref assembly for MSAL. Use #if REFERENCE_ASSEMBLY in the code files. 
+  See https://oren.codes/2018/07/09/create-and-pack-reference-assemblies-made-easy/ for details-->
```

Comment:
If you do an msbuild /t:pack, you will notice it now contains ref assemblies. Note that I haven't updated the release build yet.